web-app/django/VIM/apps/instruments/templates/instruments/detail.html

```diff
@@ -0,0 +1,167 @@
+{% extends 'base.html' %}
+
+{% load static %}
+
+{% block title %}Instrument Detail{% endblock %}
+
+{% block css_files %}
+  <link rel="stylesheet" href="{% static "instruments/css/index.css" %}" />
+  <link rel="stylesheet" href="{% static "instruments/css/detail.css" %}" />
+{% endblock %}
+
+{% block content %}
+  <div class="instrument-detail">
+    <div class="detail-header">
+      <h2>{{ instrument_name_en }}</h2>
+      <hr />
+    </div>
+    <div class="detail-body">
+      <div class="instrument-forms">
+        <table class="table table-sm table-striped table-bordered">
+          <tbody>
+            <tr>
+              <th scope="row">Wikidata ID</th>
+              <td>
+                <div class="name-form-item">
+                  <div class="name-field">
+                    <a class="view-field"
+                       href="https://www.wikidata.org/wiki/{{ wikidata_id }}"
+                       target="_blank">{{ wikidata_id }}</a>
+                    <input class="edit-field" type="text" value="{{ wikidata_id }}" />
+                  </div>
+                  <div class="button-group">
+                    <button class="btn edit">Edit</button>
+                    <button class="btn cancel">Cancel</button>
+                    <button class="btn publish">Publish</button>
+                  </div>
+                </div>
+              </td>
+            </tr>
+            <tr>
+              <th scope="row">Hornbostel-Sachs Classification</th>
+              <td>
+                <div class="name-form-item">
+                  <div class="name-field">
+                    <span class="view-field">{{ hornbostel_sachs_class }}</span>
+                    <input class="edit-field" type="text" value="{{ hornbostel_sachs_class }}" />
+                  </div>
+                  <div class="button-group">
+                    <button class="btn edit">Edit</button>
+                    <button class="btn cancel">Cancel</button>
+                    <button class="btn publish">Publish</button>
+                  </div>
+                </div>
+              </td>
+            </tr>
+            <tr>
+              <th scope="row">Classification</th>

```

Comment:
I think this should probably be "MIMO Classification" or "Musical Instrument Museums Online Classification"